plugins/rootFloxEnvs.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  self,
+  ...
+}: {
+  sourceType,
+  injectedArgs ? {},
+  dir ? sourceType,
+}: {context, ...}: let
+  floxEnvDir = context.self.outPath;
+  floxNixPath = "${floxEnvDir}/flox.nix";
+  catalogPath = "${floxEnvDir}/catalog.json";
+  namespace = ["default"];
+  result = lib.genAttrs context.systems (system: {
```

Comment:
This feels like not the capacitor way of doing this. @ysndr might have more input?

Reply:
floxEnvs/flox.nix dont comply with the capacitor interface (...) thus we have to import them explicitly